Makefile

```
@@ -472,10 +472,16 @@ deploy: prepare ## Deploy controller to the K8s cluster specified in ~/.kube/con
 	$(KUSTOMIZE) build $(CONFIG_DIR)/default | kubectl apply --namespace $(OPERATOR_NAMESPACE) -f -
 
 .PHONY: deploy-rhaii
+ifndef SKIP_IMAGE_OVERRIDES
+deploy-rhaii: apply-image-overrides
+endif
 deploy-rhaii: prepare ## Deploy controller in rhaii mode (only KServe) to the K8s cluster specified in ~/.kube/config.
```

checkmake (0.3.2)

[warning] Target "deploy-rhaii" defined multiple times (lines 476 and 478).

(uniquetargets)